mlflow/server/handlers.py

```diff
@@ -74,25 +74,30 @@ def _not_implemented():
     return response
 
 
+def _message_to_json(message):
+    return MessageToJson(message, preserving_proto_field_name=True)
```

Comment:
Can you put in a comment about the case thing -- so when code is copy/pasted they know why